src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/markup/HtmlStyle.java

```diff
@@ -75,6 +75,11 @@
     typeNameLabel,
     typeNameLink,
 
+    /**
+     * The class of the {@code pre} element presenting a snippet.
+     */
+    snippet,
+
```

Comment:
This is OK for now, given that we do not have a "public" way to process these comments at this time. However, we should probably do a cleanup later on (soon), and maybe create a new section in `HtmlStyle` for all snippet related styles ... i.e. including `bold`, `italic` etc, so that all styles end up in the upcoming documentation.